_LowLevelCode/cpp/test/combine_sqw.tests/combine_sqw.test.cpp

```diff
@@ -510,24 +510,24 @@ TEST_F(TestCombineSQW, SQW_Reader_Propagate_Pix) {
     reader.get_pix_for_bin(126, pPix_info, start_buf_pos, pix_start_num, num_bin_pix, false);
     ASSERT_EQ(pix_start_num, 334);
     ASSERT_EQ(num_bin_pix, 4);
-    for (size_t i = 0; i<num_bin_pix * 9; i++) {
-        ASSERT_EQ(pixels[pix_start_num*9+i], pix_buffer[i]);
-    }
+    // for (size_t i = 0; i<num_bin_pix * 9; i++) {
```

Comment:
Add a comment around each disabled block; there is existing commented code in Horace so we need to ensure tests aren't left disabled.
A new Horace to fix the tests should be created too.